### PR TITLE
Drop database.sql support

### PR DESCRIPTION
**Pull request by Toflar**

### CODE REVIEW

**Comment by fritzmg** [comment]
I think 4.9 should be merged upstream first (@leofeyer), since there are changes in the DcaExtractor.

**Comment by leofeyer** [comment]
4.9 has been merged upstream. Please rebase the PR now.

**Comment by Toflar** [comment]
Done.

**Review by ausi — DISMISSED** [review]

**Review by m-vo — DISMISSED** [review]

**Review by m-vo — DISMISSED** [review]

File: UPGRADE.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Version 4.* to 5.0
 
+### database.sql files
+
+Support for `database.sql` files has been dropped. Use DCA definitions and/or Doctrine DBAL schema
```
Comment (aschempp) [inline UPGRADE.md:7]: I think this should say _use Doctrine entities instead_?

**Review by aschempp — DISMISSED** [review]
LGTM except for the upgrade note 😊

File: UPGRADE.md (hunk 1 above)
Comment (Toflar) [reply UPGRADE.md:7]: No? Use whatever you want of Doctrine :)

**Review by ausi — DISMISSED** [review]

File: UPGRADE.md (hunk 1 above)
Comment (aschempp) [reply UPGRADE.md:7]: yeah but you 99.9% don't want a **DBAL schema listener**. You would either write a DCA file or a Doctrine **ORM** entity?

Comment (aschempp) [reply UPGRADE.md:7]: also btw, what we use is a ORM schema listener if I remember correctly 😅 not sure there is a DBAL listener at all.

Comment (Toflar) [reply UPGRADE.md:7]: Of course https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/events.html#schema-events. Anyway, leaving this as is.

**Review by leofeyer — APPROVED** [review]

**Comment by leofeyer** [comment]
Thank you @Toflar.

**Pull request merged**